alf/algorithms/data_transformer.py

```diff
@@ -759,6 +750,9 @@ def __init__(self,
                 exp nest.
             desired_goal_field (str): path to the desired_goal field in the
                 exp nest.
+            sparse_reward (bool): Whether to transform reward from -1/0 to 0/1.
```

Comment:
This transform from -1/0 to 0/1 seems specific to certain environments and is not a general thing.